examples/bri-3/src/bri/workgroup/worksteps/persistence/workstepsStorage.agent.ts

```diff
@@ -0,0 +1,88 @@
+import { Injectable } from '@nestjs/common';
```

Comment:
we have workgroup/agents folder already, why we dont put all agents there? this one being in persistence folder is confusing a bit @ognjenkurtic what do you think?

Reply:
Makes sense. @Kasshern if you agree please also move subjects storage agent as well.

Reply:
Moved.